src/UserProfile.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { EnumType } from 'typescript';
+
+interface userProfileProps {
+    userData: any;
+    error: string;
+}
+
+const maleDummyImage:string = "https://st2.depositphotos.com/1502311/12020/v/600/depositphotos_120206862-stock-illustration-profile-picture-vector.jpg";
```

Comment:
Instead of using a URL hosted by someone else, can we download it and add it to our directory?